pages/infrastructure_providers-validators/_meta.json

```diff
@@ -2,7 +2,7 @@
   "hardware_requirements": "Hardware Requirements",
   "required_node_configs": "Required Node Configs",
   "how_to_set_up_full_node": "Set Up a Full Node",
-  "running_full_node": "Run a Full Node",
+  "running_full_node": "Optimize Your Full Node",
```

Comment:
Updated title and reframed the page -- now about configuring an existing node for success

Reply:
should we rename the file name to reduce confusion?

Reply:
Will do in the next round of changes